<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationClass</class>
 <widget class="QWidget" name="RegistrationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Registration</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DownSampling(Voxel Grid Size):</string>
   </property>
  </widget>
  <widget class="QSlider" name="VoxelGridSize">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>301</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>211</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SIFT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>NARF</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Feature Method</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SelectFolderContainingPCDFilespushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Folder Containing PCD Files</string>
   </property>
  </widget>
  <widget class="QLabel" name="VoxelGridSizelabel">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>47</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>voxel grid size</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DownSamplepushButton">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>40</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>DownSample</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>341</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>SIFT Parameters</string>
   </property>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Minimum  Scale Size</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Surface Normals Radius</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>40</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <double>0.300000000000000</double>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>59</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>130</y>
      <width>59</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <number>3</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number Of Octaves</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="decimals">
     <number>3</number>
    </property>
    <property name="value">
     <double>0.001000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number Of Octave In Each Scale</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>381</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Correspondence Rejector Parameter</string>
   </property>
   <widget class="QSpinBox" name="spinBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>100</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>5000</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_5">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <double>0.030000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RANSAC Threshold</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Median Factor</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>RANSAC Max Iterations Run</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>351</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>KeyPoint Parameters</string>
   </property>
   <widget class="QComboBox" name="comboBox_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>PFH</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FPFH</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>191</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Feature Radius</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>62</width>
      <height>27</height>
     </rect>
    </property>
    <property name="value">
     <double>0.080000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>181</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Select KeyPoints Method</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="CalculateCorrespondencespushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>590</y>
     <width>221</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculates Correspondences</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>140</y>
     <width>611</width>
     <height>441</height>
    </rect>
   </property>
   <property name="title">
    <string>Result</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
